diktat-common/src/test/resources/test-rules-config.yml

```diff
@@ -4,163 +4,116 @@
     domainName: org.cqfn.diktat
 - name: CLASS_NAME_INCORRECT
   enabled: true
-  configuration: {}
 - name: CONSTANT_UPPERCASE
   enabled: true
-  configuration: {}
 - name: ENUM_VALUE
   enabled: true
-  configuration: {}
 - name: EXCEPTION_SUFFIX
   enabled: true
-  configuration: {}
+  configuration: { }
```

Comment:
?

Reply:
This is a yml file for tests, so I've kept different versions for configuration, which should be valid.

Reply:
good, ok